src/main/java/com/stripe/model/Invoice.java

```diff
@@ -59,6 +60,7 @@
   Long subtotal;
   Long tax;
   BigDecimal taxPercent;
+  String thresholdReason;
```

Comment:
Should this be typed as `ThresholdReason`?

Reply:
@mickjermsurawong-stripe I don't think so no. We don't do "list of constants" in java AFAIK. @ob-stripe do you agree?

Reply:
That's not what Mick meant. The `threshold_reason` attribute is an object, not a string.

Reply:
ohhh I misunderstood. That `reason` naming is throwing me off. Nice catch mick, fixing!